internal/commands/convert.go

```diff
@@ -0,0 +1,123 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/plexsystems/konstraint/internal/rego"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newConvertCommand() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "convert <dir>",
+		Short: "Convert legacy annotations to OPA Metadata Annotations",
+		Long:  "Converts legacy annotations to OPA Metadata Annotations, and does nothing on files containing OPA Metadata Annotations",
+		Example: `Convert all policies with legacy annotations to OPA Metadata annotations in-place
+
+konstraint convert examples`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path := "."
+			if len(args) > 0 {
+				path = args[0]
+			}
+
+			return runConvertCommand(path)
+		},
+	}
+
+	return &cmd
+}
+
+func runConvertCommand(path string) error {
+	regos, err := rego.GetAllSeveritiesWithoutImports(path)
+	if err != nil {
+		return fmt.Errorf("get regos: %w", err)
+	}
+
+	conversions := 0
```

Comment:
nit: In Go it is idiomatic to initialize variables with their default values rather than assigning the default value. Please update to `var conversions int`.

Reply:
agreed